ui/packages/atlasmap/src/UI/MappingTransformation.stories.tsx

```diff
@@ -52,6 +52,7 @@ export const example = () => (
     onTransformationArgumentChange={action("onActionArgumentChange")}
     onTransformationChange={action("onActionChange")}
     onRemoveTransformation={action("onRemoveTransformation")}
+    disableTransformation={false}
```

Comment:
It would be nice to use the boolean knob here to see on storybook how the component changes with the different values